arcgis-ios-sdk-samples/Scenes/Get Elevation at a point/GetElevationPointViewController.swift

```diff
@@ -0,0 +1,122 @@
+        let scene = AGSScene(basemapType: .imageryWithLabels)
+        
+        let surface = AGSSurface()
+        // Create an elevation source.
+        let elevationURL = URL(string: "https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer")
+        let elevationSource = AGSArcGISTiledElevationSource(url: elevationURL!)
+        
+        // Add the elevation source to the surface.
+        surface.elevationSources.append(elevationSource)
+        scene.baseSurface = surface
+        
+        return scene
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Add the source code button item to the right of navigation bar.
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["GetElevationPointViewController", "ElevationViewController"]
+    }
+}
+
+// MARK: - AGSGeoViewTouchDelegate
+extension GetElevationPointViewController: AGSGeoViewTouchDelegate {
+    func geoView(_ geoView: AGSGeoView, didTapAtScreenPoint screenPoint: CGPoint, mapPoint: AGSPoint) {
+        if let relativeSurfacePoint = sceneView?.screen(toBaseSurface: screenPoint) {
+//            graphicsOverlay.graphics.removeAllObjects()
+            dismiss(animated: true)
+            
+            // Create the symbol at the tapped point.
+            let marker = AGSSimpleMarkerSceneSymbol(style: .sphere, color: .red, height: 100, width: 100, depth: 200, anchorPosition: .center)
+            let point = AGSPoint(x: relativeSurfacePoint.x, y: relativeSurfacePoint.y, spatialReference: .wgs84())
+            self.graphicsOverlay.graphics.add(AGSGraphic(geometry: point, symbol: marker))
+            
+            // Get the surface elevation at the surface point.
+            self.sceneView.scene?.baseSurface!.elevation(for: relativeSurfacePoint) { (results: Double, error: Error?) in
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    self.showPopover(elevation: results, popoverPoint: screenPoint)
+                }
+            }
+        }
+    }
+
+    private func showPopover(elevation: Double, popoverPoint: CGPoint) {
+        guard let controller = storyboard?.instantiateViewController(withIdentifier: "ElevationViewController") as? ElevationViewController else {
+                return
+        }
+        // Setup the controller to display as a popover.
+        controller.modalPresentationStyle = .popover
+        controller.loadViewIfNeeded()
+        controller.elevationLabel?.text? = "Elevation at tapped point: " + (String(elevation.rounded()) + String("m"))
+        controller.popoverPresentationController?.delegate = self
+        controller.preferredContentSize = CGSize(width: 280, height: 40)
+        controller.popoverPresentationController?.passthroughViews = [sceneView as Any, navigationController?.viewControllers as Any] as? [UIView]
```

Comment:
`navigationController?.viewControllers` isn't a view, so it shouldn't be part of an array assigned to a property of type `[UIView]`. Unfortunately there isn't a way to make the back button tappable while the popover is presented.

```suggestion
        controller.popoverPresentationController?.passthroughViews = [sceneView]
```

Reply:
By getting rid of that or alternatively, including `navigationController?.navigationBar`, I am unable to tap out of/dismiss the popover. Thus, unable to tap anything in the top navigation bar. Are you sure this is what I should do?

Reply:
Yep. See [this comment](https://github.com/Esri/arcgis-runtime-samples-ios/pull/770/files#r321360902) for a solution to that problem.